docs/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Integrate Bugzilla backwards sync into the flaw and affect save (OSIDB-240)
 - Introduce Bugzilla API key as a serializer attribute (OSIDB-368)
 - Implement non-empty source validation (OSIDB-759)
+- Implement mitigation field into Flaw and update SRT notes generator (OSIDB-584)
```

Comment:
nitpick (you can ignore): Implement field sounds a bit weird to me :smiley_cat: I would say introduce or add etc.

Reply:
Thank you! Updated.